elements/range.lua

```diff
@@ -45,11 +45,7 @@ local function Update(self, event)
 	local connected = UnitIsConnected(unit)
 	if(connected) then
 		inRange, checkedRange = UnitInRange(unit)
-		if(checkedRange and not inRange) then
-			self:SetAlpha(element.outsideAlpha)
-		else
-			self:SetAlpha(element.insideAlpha)
-		end
+		self:SetAlphaFromBoolean(inRange, element.insideAlpha, element.outsideAlpha)
```

Comment:
`UnitInRange` returns `false` `false` when not in a group, so the target unit appears always out of range this way.

I would have suggested

```suggestion
		self:SetAlphaFromBoolean(checkedRange and not inRange, element.outsideAlpha, element.insideAlpha)
```

but it results in `attempt to perform boolean test on local 'checkedRange' (a secret value)`.